config.php

```diff
@@ -52,6 +53,11 @@
             'event' => 'memberRemoved',
             'callback' => ['humhub\modules\ethereum\calls\Space', 'leaveSpace']
         ],
+        [
+            'class' => OverviewController::class,
```

Comment:
this is the OverviewController from the other module?

Reply:
yes from the *xcoin-module*